packages/server/README.md

```diff
@@ -0,0 +1,20 @@
+## How to add a new route
+
+1. Add a method to the controller with `@Get` or other http method decorator. To get request body, path params, query params, etc, use decorators on the arguments. [Available decorators](https://docs.nestjs.com/controllers#request-object). Additionally, we have custom decorators `@User` and `@UserId` to get the currently logged in user.
+2. If a new controller is needed, use `yarn nest g controller <CONTROLLERNAME>` to auto-generate one. 
+3. **IMPORTANT** If route/controller is authenticated: `@UseGuards(JwtAuthGuard)`
+4. If you want to use database model serialization (`@Expose @Exclude` stuff): `@UseInterceptors(ClassSerializerInterceptor)`
+
+## How to access the admin panel
+
+1. `yarn cli create:admin <username>` then add your password through stdin.
```

Comment:
We should add that you have to execute this command from within `/packages/server`, Will and I both are dumbo

Also when i did `yarn cli create` before starting the server the user wasn't added, so maybe add a bit saying that you have to start the app first?

Reply:
I will add a top level package json script too that'll work the way `dev:db:seed` does

Reply:
if DB is up, it should work. server should not be needed